javascript/todo/src/controls.js

```diff
@@ -0,0 +1,21 @@
+export default class Controls {
+  #titles;
+  #dis;
+  constructor(title, discription) {
+    this.#titles = title;
+    this.#dis = discription;
+  }
+
+  getTitle() {
+    return console.log(this.#titles);
+  }
+  setTitle(title) {
+    this.#titles = title;
+  }
+  getDis() {
+    return console.log(this.#dis);
+  }
+  setDis(discription) {
```

Comment:
Try to write more understandable method names:
recommended: 
setDis(discription) => setDescription(newDescription)